es6/tests/utils.js

```diff
@@ -543,6 +543,15 @@ function createDoc(name) {
 	return loadDocument(name, documentCache[name].loadedContent);
 }
 
+function createDocV4(name, { modules }) {
+	const zip = documentCache[name].getZip();
```

Comment:
Can you use `const zip = getZip(name)` here ?

Reply:
Yup, will do it.